lefthook.yaml

```diff
@@ -0,0 +1,14 @@
+commit-msg:
```

Comment:
shall we bring this block below pre commit, usually it runs after the pre commit hook i guess, the order here has no effect on the final order it's run but it seems nicer to keep them in execution order

Reply:
Done